src/components/my-journey/Connector.tsx

```diff
@@ -1,3 +1,4 @@
+
 import React from "react";
 interface ConnectorProps {
   fill?: boolean;
```

Comment:
`line 4 ` please change the variable names to isFill since it is a boolean variable.